spec.html

```diff
@@ -65,41 +65,33 @@ <h1>Import Calls</h1>
       <emu-clause id="sec-import-call-runtime-semantics-evaluation">
         <h1>Runtime Semantics: Evaluation</h1>
 
-        <emu-grammar>ImportCall : `import` `(` AssignmentExpression `,`? `)`</emu-grammar>
-        <emu-alg>
-          1. Let _referencingScriptOrModule_ be ! GetActiveScriptOrModule().
-          1. Let _argRef_ be the result of evaluating |AssignmentExpression|.
-          1. Let _specifier_ be ? GetValue(_argRef_).
-          1. Let _promiseCapability_ be ! NewPromiseCapability(%Promise%).
-          1. Let _specifierString_ be ToString(_specifier_).
-          1. IfAbruptRejectPromise(_specifierString_, _promiseCapability_).
-          1. <ins>Let _moduleRequest_ be a new ModuleRequest Record { [[Specifier]]: _specifierString_, [[Assertions]]: an empty List }.</ins>
-          1. Perform ! HostImportModuleDynamically(_referencingScriptOrModule_, <del>_specifierString_,</del> <ins>_moduleRequest_,</ins> _promiseCapability_).
-          1. Return _promiseCapability_.[[Promise]].
-        </emu-alg>
-
-        <ins><emu-grammar>ImportCall : `import` `(` AssignmentExpression `,` AssignmentExpression `,`? `)`</emu-grammar></ins>
+        <ins><emu-grammar>ImportCall : `import` `(` AssignmentExpression `,`? AssignmentExpression? `,`? `)`</emu-grammar></ins>
```

Comment:
Is it correct to combine the two syntaxes together like this here? When everything is opt at once like this, these would be valid syntax:
```
import(foo, ,);
import(foo bar);
```